### ISSUE TITLE
Error on kali

### ISSUE THREAD

**Issue by legnaro** [guh/nymea-networkmanager#7]
Hi everybody,

I have dificults to run on kali linux.

When I run manually: nymea-networkmanager  --mode always -d  

I have:
I | Application: Networkmanager networking is now enabled
 I | NetworkManager: Wireless networking enabled
 I | Application: Networkmanager wireless networking is now enabled
 I | NetworkManager: [+] NetworkDevice("lo" - "Generic", "NetworkDeviceStateUnmanaged")
 I | NetworkManager: [+] WirelessNetworkDevice("wlan0", "B8:27:EB:D7:6B:1E", 0 [Mb/s], "NetworkDeviceStateUnmanaged") 
 I | NetworkManager: The network manager is now available
 I | Application: Networkmanager is now available.
 I | Application: Start the bluetooth service because of "always" mode.
 I | Application: Start service
 W | NymeaService: Could not enable/disable bluetooth hardware resource. D-Bus interface not available.
 I | Application: Start bluetooth service
 I | NetworkManager: Network manager initialized successfully.
 I | Application: Start the bluetooth service because of "always" mode.
 I | Application: Start service
 W | NymeaService: Could not enable/disable bluetooth hardware resource. D-Bus interface not available.
 I | Application: Start bluetooth service
 I | BluetoothServer: -------------------------------------
 I | BluetoothServer: Starting bluetooth server
 I | BluetoothServer: -------------------------------------
 I | BluetoothServer: Local device "kali" "B8:27:EB:28:94:E1"
 W | qt.bluetooth: Using BlueZ kernel ATT interface
 I | BluetoothServer: Create NetworkService.
 I | BluetoothServer: NetworkService: Notify state changed "\x07"
 I | BluetoothServer: NetworkService: Notify networking enabled changed: enabled
 I | BluetoothServer: NetworkService: Notify wireless networking enabled changed: enabled
 I | BluetoothServer: Create WirelessService.
 I | BluetoothServer: Start advertising "Inove Media Meter" "B8:27:EB:28:94:E1"
 I | BluetoothServer: Controller state advertising...
 I | Application: Bluetooth server started

The first error is:
invalid D-BUS hardmanager interface.

how could I fix this?

**Comment by t-mon** [comment]
Hi! 
The warning `W | NymeaService: Could not enable/disable bluetooth hardware resource. D-Bus interface not available.` comes from the nymea service, which is meant to disable the Bluetooth client connections in the IoT server `nymead` when the nymea-networkmanager is starting the Bluetooth server. With Bluetooth 4.0 it is not possible to have a BTLE client and server running at the same time. Since the `nymead` is not installed in the default Berrylan image, this warning can be ignored. It shouldn't have any impact on the network setup functionality.

**Comment by legnaro** [comment]
Hi t-mon, tks for your quick answer !
You are rigth, after this warning I was able to connect with berrylan app and send my WiFi credentials, I can see in the log the message with user and pass. The problem is after send the credentials I got others warnings (I thought it was related with the first warning), apparently this warning affected the wifi configuration:
![img_20181224_161336106](https://user-images.githubusercontent.com/7379954/50422353-2a715180-0831-11e9-960b-b938d05fcc40.jpg)
Image attached.

Apparently the system can't add new connection. Im not sure how can I fix this.

**Comment by legnaro** [comment]
Apparently the same warnings.

Just to guarantee that's update occurs correctly, follow the details version:

apt-cache show nymea-networkmanager
```
Package: nymea-networkmanager
Version: 0.2.0+201810301849~c457104~stretch
Architecture: armel
Maintainer: Simon Stürz
Installed-Size: 224
Depends: init-system-helpers (>= 1.18~), libqt5network5, libqt5bluetooth5, network-manager, libnymea-networkmanager (= 0.2.0+201810301849~c457104~stretch)
Priority: optional
Section: utils
Filename: pool/main/n/nymea-networkmanager/nymea-networkmanager_0.2.0+201810301849~c457104~stretch_armel.deb
Size: 58140
SHA256: 0156d75ee60baacb33a58c7ff5f9fb6b84354d3ecf6302ccea120df5e464f076
SHA1: 1200b64c7d518fd88dd02d38a6147bcfc069829f
MD5sum: 1c4ea929cf2b8b2847e74d39f8001572
Description: Daemon for wireless configuration using bluetooth LE.
  This daemon allows to configure a wireless network using a bluetooth
  low energy gatt server. The tool is written in Qt 5.
Description-md5: 88947fde66bdb56e42c662b3d6051da1
```
If you prefer I can try to open a SSH connection for you in a public IP (I can send the pass via email)

tks

**Comment by t-mon** [comment]
Sorry for the late respone! 
The new version should be available soon (`0.3.2`) and should fix this issue. It would be great if you could verify! Thanks :)

**Comment by legnaro** [comment]
Great version. It works really well! Thanks for your support.
